packages/vite-plugin-svelte/src/utils/id.js

```diff
@@ -50,7 +50,8 @@ function parseToSvelteRequest(id, filename, rawQuery, root, timestamp, ssr) {
 		query,
 		timestamp,
 		ssr,
-		raw
+		raw,
+		inline: !!query.inline
```

Comment:
raw and direct are handled above, is inline distinct to warrant its own flag? query is returned above too so the consumer could also use query.inline

Reply:
I updated to use `svelteRequest.query.inline` instead. I initially saw `raw` and was trying to map the same pattern here.